xml/System.Xaml/XamlLanguage.xml

```diff
@@ -989,8 +989,8 @@
         <ReturnType>System.Xaml.XamlType</ReturnType>
       </ReturnValue>
       <Docs>
-        <summary>Gets a <see cref="T:System.Xaml.XamlType" /> for the <see langword="Single" /> XAML language intrinsic.</summary>
-        <value>A <see cref="T:System.Xaml.XamlType" /> for the <see langword="Single" /> XAML language intrinsic.</value>
+        <summary>Gets a <see cref="T:System.Xaml.XamlType" /> for the <see cref="T:System.Single" /> XAML language intrinsic.</summary>
+        <value>A <see cref="T:System.Xaml.XamlType" /> for the <see cref="T:System.Single" /> XAML language intrinsic.</value>
```

Comment:
I think these two may also ve referring to something other than `T:System.Single`.

Reply:
There are referring to the `x:Single` type in XAML, which maps to `System.Single` in .NET but they are different things. I'm changing these to `<code>`.